lib/src/models/twitch_clip.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'twitch_clip.freezed.dart';
+part 'twitch_clip.g.dart';
+
+@freezed
+class TwitchClip with _$TwitchClip {
+  const factory TwitchClip({
+    @JsonKey(name: 'id') required String id,
+    @JsonKey(name: 'url') required String url,
+    @JsonKey(name: 'embed_url') required String embedUrl,
+    @JsonKey(name: 'broadcaster_id') required String broadcasterId,
+    @JsonKey(name: 'broadcaster_name') required String broadcasterName,
+    @JsonKey(name: 'creator_id') required String creatorUd,
+    @JsonKey(name: 'creator_name') required String creatorName,
+    @JsonKey(name: 'video_id') required String videoUd,
+    @JsonKey(name: 'game_id') required String gameUd,
+    @JsonKey(name: 'language') required String language,
+    @JsonKey(name: 'title') required String title,
```

Comment:
`@JsonKey` is not needed:
```suggestion
    required String language,
    required String title,
```